src/NetworkTest/testConnectivity/index.ts

```diff
@@ -338,7 +346,17 @@ export function testConnectivity(
           failedTests,
           success: false,
         };
-        otLogging.logEvent({ action: 'testConnectivity', variation: 'Success' });
+        otLogging.logEvent({
+          action: 'testConnectivity',
+          variation: 'Failure',
+          payload: {
+            failedTests: failedTests.map(test => ({
+              type: test.type,
+              error: test.error.name,
+            })),
+            errorNames: errors.map(e => e.name),
```

Comment:
@behei-vonage should we consider Undefined error Name and just to log `Unknown Error` ? Same for others failures cases.